aspnetcore/security/anti-request-forgery.md

```diff
@@ -1,18 +1,311 @@
+```
+
+You can also explicitly add an antiforgery token to a ``<form>`` element you create without using tag helpers or HTML helpers by using ``@Html.AntiForgeryToken()``:
+
+<!-- literal_block {"xml:space": "preserve", "language": "html", "dupnames": [], "linenos": false, "classes": [], "ids": [], "backrefs": [], "highlight_args": {}, "names": []} -->
+
+```html
+<form action="/" method="post">
+    @Html.AntiForgeryToken()
+</form>
+```
+
+<!-- literal_block {"xml:space": "preserve", "language": "html", "dupnames": [], "linenos": false, "classes": [], "ids": [], "backrefs": [], "highlight_args": {}, "names": []} -->
+
+```html
+In each of the above cases, ASP.NET Core will add a hidden form field like the following:
+
+<input name="__RequestVerificationToken" type="hidden" value="CfDJ8NrAkSldwD9CpLRyOtm6FiJB1Jr_F3FQJQDvhlHoLNJJrLA6zaMUmhjMsisu2D2tFkAiYgyWQawJk9vNm36sYP1esHOtamBEPvSk1_x--Sg8Ey2a-d9CV2zHVWIN9MVhvKHOSyKqdZFlYDVd69XYx-rOWPw3ilHGLN6K0Km-1p83jZzF0E4WU5OGg5ns2-m9Yw" />
+```
+
+If you don't want to include an antiforgery token as part of your generated form, you can disable the behavior using ``asp-antiforgery="false"``:
+
+<!-- literal_block {"xml:space": "preserve", "language": "html", "dupnames": [], "linenos": false, "classes": [], "ids": [], "backrefs": [], "highlight_args": {}, "names": []} -->
+
+```html
+<form asp-controller="Manage" asp-action="ChangePassword" 
+  method="post" asp-antiforgery="false">
+
+</form>
+```
+
+ASP.NET Core includes three [filters](../mvc/controllers/filters.md) for working with antiforgery tokens: ``ValidateAntiForgeryToken``, ``AutoValidateAntiforgeryToken``, and ``IgnoreAntiforgeryToken``.
+
+### ValidateAntiForgeryToken
+
+The ``ValidateAntiForgeryToken`` is an action filter that can be applied to an individual action, a controller, or globally for the app. Requests made to actions that have this filter applied will be blocked unless the request includes a valid antiforgery token.
+
+<!-- literal_block {"xml:space": "preserve", "language": "c#", "dupnames": [], "linenos": false, "classes": [], "ids": [], "backrefs": [], "highlight_args": {}, "names": []} -->
+
+```c#
+[HttpPost]
+[ValidateAntiForgeryToken]
+public async Task<IActionResult> RemoveLogin(RemoveLoginViewModel account)
+{
+    ManageMessageId? message = ManageMessageId.Error;
+    var user = await GetCurrentUserAsync();
+    if (user != null)
+    {
+        var result = await _userManager.RemoveLoginAsync(user, account.LoginProvider, account.ProviderKey);
+        if (result.Succeeded)
+        {
+            await _signInManager.SignInAsync(user, isPersistent: false);
+            message = ManageMessageId.RemoveLoginSuccess;
+        }
+    }
+    return RedirectToAction(nameof(ManageLogins), new { Message = message });
+}
+```
+
+The ``ValidateAntiForgeryToken`` attribute will require a token for every request, including *GET* requests. If you apply it broadly, you can override it with the ``IgnoreAntiforgeryToken`` attribute.
```

Comment:
If you wanted to make sure it wasn't forgotten. However, the AutoValidateAntiforgeryToken is designed for this scenario, since it is already smart enough to ignore GET and other requests.